src/core/server/rendering/rendering_service.tsx

```diff
@@ -164,12 +148,140 @@ export class RenderingService {
 
   public async stop() {}
 
+  // assign values for branding config, return a BrandingAssignment type object
```

Comment:
nit: can this functions have proper jsdoc commenting.

Reply:
Changed all inline comments to jsdoc comment format. Thx!